Survey/Sources/Domain/UseCases/LogInUseCase.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  LogInUseCase.swift
+//  Survey
+//
+//  Created by Khanh on 01/08/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Combine
+import Foundation
+
+protocol LogInUseCaseType {
```

Comment:
For protocol, I think we don't need type in name

```suggestion
protocol LogInUseCaseProtocol {
```